srcpkgs/grub-btrfs/template

```diff
@@ -9,17 +9,23 @@ maintainer="Anjandev Momi <anjan@momi.ca>"
 license="GPL-3.0-or-later"
 homepage="https://github.com/Antynea/grub-btrfs"
 distfiles="https://github.com/Antynea/grub-btrfs/archive/v${version}.tar.gz"
-checksum=62461133c697643e98a608239773f500edee3e4055c5729fef6639d1cd610243
+checksum=3ef53f16a7bc0fdd5a5ac2216a41bf1b2f4fd2283717c5c240415d5d24d3734b
 
 post_install() {
 	rm -rf -- "${DESTDIR}"/usr/lib/systemd
 }
 
-grub-btrfs-runit_package() {
+runit-grub-btrfs_package() {
 	depends="wendy ${sourcepkg}>=${version}_${revision}"
 	short_desc+=" - runit service"
 
 	pkg_install() {
 		vsv grub-btrfs
 	}
 }
+
+grub-btrfs-runit_package() {
+	depends="runit-grub-btrfs>=${version}_${revision}"
+	short_desc+=" - runit service (transitional dummy package)"
+	build_style=meta
+}
```

Comment:
Unless you know someone who couldn't find the package because of the naming, I'd rather not add one more symlink to keep track of here.

Reply:
So should I just replace the old package with the new one and add it to removed-package? Or keeping the package's name as it is right now?
I don't know anyone having problems with the naming. It is just for the sake of consistent naming scheme.

Reply:
Keeping as it is right now.

I appreciate trying to improve it, but i think in this case the lack of churn wins out :)

Reply:
I reverted it back